Detectors/FIT/FT0/calibration/testWorkflow/FT0CalibCollectorWriterSpec.h

```diff
@@ -35,7 +35,7 @@ class FT0CalibCollectorWriter : public o2::framework::Task
  public:
   void createAndOpenFileAndTree()
   {
-    TString filename = TString::Format("collFT0_%d.root", mCount);
+    TString filename = TString::Format("collFT0.root", mCount);
```

Comment:
Is this going to work? I mean without %d in the name, the mCount is meaningless, right?

Reply:
just forgot to return